test/test_helper.rb

```diff
@@ -44,6 +44,8 @@ class ActiveSupport::TestCase
     DatabaseCleaner.clean
   end
 
+  parallelize(workers: :number_of_processors)
```

Comment:
System tests in circle fight this pretty hard - lots of selenium errors. Unit tests work marvelously though.